README.md

```diff
@@ -56,6 +56,11 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `projectName` - The preferred project in which the debugger searches for classes. There could be duplicated class names in different projects. This setting also works when the debugger looks for the specified main class when launching a program.
 - `cwd` - The working directory of the program.
 - `env` - The extra environment variables for the program.
+- `stopOnEntry` - Automatically stop program after launch.
+- `console` - Where to launch the program. Defaults to `internalConsole`.
```

Comment:
Is this term console provide by VSCode?  Normally, in this case, it will call  as `shell`?

Reply:
reference to vscode-node-debug implemented by vscode team, they used the term `console`.

Reply:
The specified console to launch the program. Defaults to ...